packages/attestation-service/src/requestHandlers/get_attestation.ts

```diff
@@ -87,10 +96,35 @@ class GetAttestationRequestHandler {
 }
 
 export async function handleGetAttestationRequest(
-  _req: express.Request,
+  req: express.Request,
   res: express.Response,
   getRequest: GetAttestationRequest
 ) {
+  const {
+    headers: { authentication },
+  } = req
+  if (!authentication) {
```

Comment:
Is it enough to put it behind the same flag `if (getRequest.securityCode)`? My feeling is yes, there won't be any security code requests without the header

Reply:
Hmm, that is a good point. As long as we ensure that wallet doesn't ramp the feature before adding the header you are right.

Since we've put that feature currently as a blocker for launch (mostly to prevent issues with backwards compatibility where old version of wallet will be missing headers), I think your solution works 👍

Reply:
(cc @timmoreton if you see any other potential issues here)

Reply:
We should have this behind a config flag, false by default. We said we want to roll out security codes without the signature when everyone is on 1.1.0 and TLS. I think that should still be the plan. Having a config flag here means we won't have to force upgrade all Valora users until everyone is on 1.2.0 and we want to enable this flag. Otherwise we have to force upgrade all Valora users before anyone can upgrade to 1.2.0.

Reply:
Wallets under the impression that they are going to make the change before ramping. Since they need to push a new version anyway to "activate" the feature.